libraries/stdlib/src/kotlin/text/Char.kt

```diff
@@ -37,3 +37,17 @@ public fun Char.equals(other: Char, ignoreCase: Boolean = false): Boolean {
  * Returns `true` if this character is a Unicode surrogate code unit.
  */
 public fun Char.isSurrogate(): Boolean = this in Char.MIN_SURROGATE..Char.MAX_SURROGATE
+
+/**
+ * Minimum value for character
+ * @see <a href="https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html">
```

Comment:
I don't think this way of linking is supported in `@see` tag.  I'm not sure what is the purpose of these links anyway.

Reply:
Is it good now?